python/lsst/meas/base/forcedPhotImage.py

```diff
@@ -143,6 +140,23 @@ def runDataRef(self, dataRef, psfCache=None):
         if psfCache is not None:
             exposure.getPsf().setCacheSize(psfCache)
         refCat = self.fetchReferences(dataRef, exposure)
+
+        measCat = self.run(exposure, refCat, refWcs, dataRef)
```

Comment:
This is a struct containing measCat, so this code looks wrong. I think you want something like this:
```
runResult = self.run(...)
self.writeOutput(dataRef, runResult.measCat)
```